src/package.xml

```diff
@@ -1465,23 +1465,25 @@
         <members>Payment_Services_Configuration__c.Service_Key__c</members>
         <members>Payment_Services_Configuration__c.Service__c</members>
         <members>Payment_Services_Configuration__c.Value__c</members>
-        <members>RecurringDonationChangeHistory__c.Change_Type__c</members>
-        <members>RecurringDonationChangeHistory__c.Effective_Date__c</members>
-        <members>RecurringDonationChangeHistory__c.New_Amount__c</members>
-        <members>RecurringDonationChangeHistory__c.New_Campaign__c</members>
-        <members>RecurringDonationChangeHistory__c.New_Payment_Method__c</members>
-        <members>RecurringDonationChangeHistory__c.New_Schedule__c</members>
-        <members>RecurringDonationChangeHistory__c.New_Status_Reason__c</members>
-        <members>RecurringDonationChangeHistory__c.New_Status__c</members>
-        <members>RecurringDonationChangeHistory__c.New_Yearly_Value__c</members>
-        <members>RecurringDonationChangeHistory__c.Previous_Amount__c</members>
-        <members>RecurringDonationChangeHistory__c.Previous_Campaign__c</members>
-        <members>RecurringDonationChangeHistory__c.Previous_Payment_Method__c</members>
-        <members>RecurringDonationChangeHistory__c.Previous_Schedule__c</members>
-        <members>RecurringDonationChangeHistory__c.Previous_Status_Reason__c</members>
-        <members>RecurringDonationChangeHistory__c.Previous_Status__c</members>
-        <members>RecurringDonationChangeHistory__c.Previous_Yearly_Value__c</members>
-        <members>RecurringDonationChangeHistory__c.Recurring_Donation__c</members>
+        <members>RecurringDonationChangeHistory__c.ChangeType__c</members>
+        <members>RecurringDonationChangeHistory__c.EffectiveDate__c</members>
+        <members>RecurringDonationChangeHistory__c.NewAmount__c</members>
+        <members>RecurringDonationChangeHistory__c.NewCampaign__c</members>
+        <members>RecurringDonationChangeHistory__c.NewInstallmentFrequency__c</members>
```

Comment:
I think this object is missing the DayOfMonth field. That's part of the schedue.

Reply:
We decided not to track that field, this object is too big already (part of the reason we were going to concatenate schedule into a single field)